### PR TITLE
Calendar UI basics added

### PR DESCRIPTION
**Pull request by peter-asztalos**

### CODE REVIEW

File: src/hooks/scheduledEvent/useScheduledEvents.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      ScheduledEvent,
+      'id' | 'bookingType' | 'startsAt' | 'endsAt' | 'description'
+    >[]
+  >([]);
+
+  const unauthorizedApi = useUnauthorizedApi();
+
+  const refresh = useCallback(() => {
+    setLoading(true);
+    unauthorizedApi()
```
Comment (jekabs-karklins) [inline src/hooks/scheduledEvent/useScheduledEvents.ts:19]: I presume we will use authorizedApi latter :)

**Review by jekabs-karklins — APPROVED** [review]

File: src/hooks/scheduledEvent/useScheduledEvents.ts (hunk 1 above)
Comment (peter-asztalos) [reply src/hooks/scheduledEvent/useScheduledEvents.ts:19]: I hope so :D

File: src/components/Calendar/Calendar.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
+import { Grid, makeStyles } from '@material-ui/core';
+import clsx from 'clsx';
+import generateScheduledEventFilter from 'filters/scheduledEvent/scheduledEventsFilter';
+import moment from 'moment';
```
Comment (martin-trajanovski) [inline src/components/Calendar/Calendar.tsx:4]: @peter-asztalos I see you are using here moment and in the backend luxon. Is that for some reason or we can use luxon on the fontend as well?

**Review by martin-trajanovski — APPROVED** [review]

File: src/components/Calendar/Calendar.tsx (hunk 2 above)
Comment (peter-asztalos) [reply src/components/Calendar/Calendar.tsx:4]: The calendar library provides localization using different kind of libs, moment is one of them currently, so I decided on moment as I think many of us is already familiar with it's API and provides wide range of function for other uses cases also.

**Pull request merged**